bigbluebutton-config/bin/bbb-record

```diff
@@ -45,20 +45,31 @@
 #set -e
 #set -x
 
-BASE=/var/bigbluebutton/recording
-STATUS=$BASE/status
 source /etc/bigbluebutton/bigbluebutton-release
 
 BBB_USER=bigbluebutton
-BBB_WEB=$(cat /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties | sed -n '/^bigbluebutton.web.serverURL/{s/.*\///;p}')
-PLAYBACK_PROTOCOL=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(playback_protocol\)/{s/.*playback_protocol:[ ]*//;s/;//;p}')
-RECORDING_DIR=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(recording_dir\)/{s/.*recording_dir:[ ]*//;s/;//;p}')
-LOG_DIR=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(log_dir\)/{s/.*log_dir:[ ]*//;s/;//;p}' | head -1)
-PUBLISHED_DIR=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(published_dir\)/{s/.*published_dir:[ ]*//;s/;//;p}')
-RAW_AUDIO_SRC=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(raw_audio_src\)/{s/.*raw_audio_src:[ ]*//;s/;//;p}')
-RAW_VIDEO_SRC=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(raw_video_src\)/{s/.*raw_video_src:[ ]*//;s/;//;p}')
-RAW_SCREENSHARE_SRC=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(raw_screenshare_src\)/{s/.*raw_screenshare_src:[ ]*//;s/;//;p}')
-RAW_PRESENTATION_SRC=$(cat /usr/local/bigbluebutton/core/scripts/bigbluebutton.yml | sed -n '/\(raw_presentation_src\)/{s/.*raw_presentation_src:[ ]*//;s/;//;p}')
+BBB_PROPERTIES=/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+BBB_YML=/usr/local/bigbluebutton/core/scripts/bigbluebutton.yml
+
+SERVER_URL=$(sed -n '/^bigbluebutton.web.serverURL/{s/.*\///;p}' < $BBB_PROPERTIES)
+
+# $1: Desired property
+# Only returns 'PRODUCTION' values
+get_yml_value(){
+   sed -n '/\('"$1"'\)/{s/.*'"$1"':[ ]*//;s/;//;p}' < $BBB_YML | head -1
```

Comment:
We require that the `yq` tool is installed already for other bigbluebutton scripts, we should probably use that here as well.

The get_yml_value() function could just be:
`yq read "$BBB_YML" "$1"`

Reply:
Good idea, was unaware of that tool.